tests/test_account_chart.py

```diff
@@ -6,7 +6,6 @@
 import requests
```

Comment:
We're missing e test that ensures adding/updating an account with `vat_code=None` works (and assigns no no VAT code).

Reply:
I guess this part is covered in `test_account_mutators` test suit

1. We're adding the account with `None` Vat code
```python
    new_account = {
        'account': 1146,
        'currency': 'CHF',
        'text': 'test create account',
        'vat_code': None,
        'group': '/Assets/Anlagevermögen'
    }
    cashctrl_ledger.add_account(**new_account)
```
2.  Updating that Vat code with existent one
```python
    new_account = {
        'account': 1146,
        'currency': 'CHF',
        'text': 'test update account',
        'vat_code': 'TestCodeAccounts',
        'group': '/Assets/Anlagevermögen'
    }
    cashctrl_ledger.update_account(**new_account)
```


And the same we do with update part:
```python
    new_account = {
        'account': 1145,
        'currency': 'USD',
        'text': 'test update account without VAT',
        'vat_code': None,
        'group': '/Assets/Anlagevermögen'
    }
    cashctrl_ledger.update_account(**new_account)
```

Isn`t this not enough for that?

Reply:
Thanks. I had somehow missed this.